test/Test/Main.purs

```diff
@@ -1,24 +1,26 @@
 module Test.Main where
 
 import Prelude
+import Effect (Effect)
 import Data.HeytingAlgebra (ff, tt, implies)
 import Data.Ord (abs)
 import Test.Data.Generic.Rep (testGenericRep)
-import Test.Utils (AlmostEff, assert)
+import Test.Utils (assert)
 
-main :: AlmostEff
+main :: Effect Unit
 main = do
-    testNumberShow show
+    -- testNumberShow show
```

Comment:
Why is this commented?

Reply:
I left for another PR. Partly because I didn't know how to implement it yet. Should I somehow properly leave a TODO or make an issue? I wanted comment it out so I could run the test suite.

Reply:
Ah that makes sense